### ISSUE TITLE
ihu

### ISSUE THREAD

**Issue by Reaper10** [TylerOderkirk/ihu#1]
is I hear u dead?

**Comment by jose1711** [comment]
not necessarily. i was able to compile ihu on an up-to-date arch linux with patches downloaded from https://sourceforge.net/p/ihu/feature-requests/22/. will fork this and merge them into the tree

**Comment by jose1711** [comment]
https://github.com/jose1711/ihu contains patches enabling compilation with qt4 which you may still find around
